src/compiler/transformers/component-native/native-constructor.ts

```diff
@@ -32,15 +33,17 @@ export const updateNativeConstructor = (
   if (cstrMethodIndex >= 0) {
     // add to the existing constructor()
     const cstrMethod = classMembers[cstrMethodIndex] as ts.ConstructorDeclaration;
+    // a constructor may not have a body (e.g. in the case of constructor overloads)
+    const cstrBodyStatements: ts.NodeArray<ts.Statement> = cstrMethod.body?.statements ?? ts.factory.createNodeArray();
```

Comment:
This function largely _assumes_ that there won't be any overloaded ctor's in a Stencil component. While I don't know if that will always hold true, I don't think it's something we need to worry about this second.